skimage/skimage_lbp_simple.py

```diff
@@ -12,26 +12,25 @@
 import optuna
 
 import skimage.feature as ft
-from sklearn.datasets import fetch_olivetti_faces
+from sklearn.datasets import load_digits
```

Comment:
[Notes] `load_digits` is used in the dask example.
https://github.com/optuna/optuna-examples/blob/b66607c7bae14aadf0e371a36cdabab00a13dcdc/dask/dask_simple.py#L18